app/src/main/java/com/github/multimatum_team/multimatum/MainActivity.kt

```diff
@@ -26,7 +27,8 @@ class MainActivity : AppCompatActivity() {
     @Inject
     lateinit var deadlineRepository: DeadlineRepository
 
-    private lateinit var viewModel: DeadlineListViewModel
+    private val viewModel: DeadlineListViewModel by viewModels()
```

Comment:
Why is the `by viewModel()` necessary ?

Reply:
It is not strictly necessary for this PR but since I was working on the viewmodel I refactored the activities to inject the viewmodel.
This will allow us to test the activities by injecting a mock viewmodel.